.github/workflows/ci.yml

```diff
@@ -55,8 +55,3 @@ jobs:
 
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v1
-        with:
-          # The codecov token to upload report. Can be retrieved from your Codecov dashboard.
-          # This should be set from `GitHub Repo > Settings > Secrets`.
-          token: ${{ secrets.CODECOV_TOKEN }}
-          fail_ci_if_error: true
```

Comment:
should we then enable this again?

Reply:
We don't need token. I think because it was failing before and I wasn't sure of the reason, I thought having token will help. But I see the upload to CodeCov is working fine without token.

On `fail_ci_if_error`, I think we can skip it too. The more important thing is the test cases shouldn't fail. Additionally, now we've removed token and our repo will be treated as open-source, I don't think upload to codecov will fail ever.